internal/api/server.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+
+	"github.com/redhatinsights/ros-ocp-backend/internal/logging"
+	"github.com/redhatinsights/ros-ocp-backend/internal/config"
+
+)
+
+func StartAPIServer() {
+	app := echo.New()
+	log := logging.GetLogger()
+	cfg := config.GetConfig()
+	app.Use(middleware.Logger())
+	app.Use(middleware.CORSWithConfig(middleware.CORSConfig{
+		AllowMethods: []string{http.MethodGet},
+	}))
+
+	app.GET("/api/cost-management/v1/recommendations/openshift/", GetRecommendationSetList)
+	app.GET("/api/cost-management/v1/recommendations/openshift/:recommendation_id/", GetRecommendationSet)
```

Comment:
We will need to check with @chambridge how reverse proxy thing would be working and based on that decide the API path of ros-ocp side. Because path `/api/cost-management` will be already reserved for in 3scale and will be pointing to the service in cost-management namespace in openshift cluster.

Reply:
@chambridge Also do we need the trailing slashes for both the endpoints?
These are present in the routes on the Architecture doc.

Reply:
They are not required. This is just a Django-ism so I'm use to writing APIs with the trailing slash.